docs/core/integration-quality-scale/index.md

```diff
@@ -0,0 +1,176 @@
+
+From a technical perspective, the integration needs to have full automated test coverage of its codebase to ensure the set integration quality is maintained now and in the future.
+
+All integrations that have devices in the Works with Home Assistant program are at least required to have this tier.
+
+Characteristics:
+- Provides everything “Silver” has.
+- Has the best end-user experience an integration can offer; streamlined and intuitive.
+- Can be automatically discovered, simplifying the integration setup.
+- Integration can be reconfigured and adjusted.
+- Supports translations.
+- Extensive documentation, aimed at non-technical users.
+- It supports updating the software/firmware of devices through Home Assistant when possible.
+- Provides the tooling and documentation troubleshooting issues.
+- The integration has automated tests covering the entire integration.
+- Required level for integrations providing devices in the Works with Home Assistant program.
+
+### 🏆 Platinum
+Platinum is the highest tier an integration can reach, the epitome of quality within Home Assistant. It not only provides the best user experience but also achieves technical excellence by adhering to the highest standards, supreme code quality, and well-optimized performance and efficiency.
+
+Characteristics:
+- Provides everything “Gold” has.
+- All source code follows all coding and Home Assistant integration standards and best practices and is fully typed with type annotations and clear code comments for better code clarity and maintenance.
+- A fully asynchronous integration code base ensures efficient operation.
+- Implements efficient data handling, reducing network and CPU usage.
+
+
+
+## Adjusting the tier of an integration
+When an integration reaches the minimum requirements for a certain tier, a contributor can open a pull request to adjust the scale for the integration.
+This request needs to be accompanied by the full checklist for each rule of scale (including all rules of lower tiers), demonstrating that it has met those requirements.
+The checklist can be found [here](checklist).
+
+Once the Home Assistant core team reviews and approves it, the integration will display the new tier as of the next major release of Home Assistant.
+
+Besides upgrading integration to a higher tier on the scale, it is also possible for an integration to be downgraded to a lower tier.
+This can, for example, happen when there is no longer an active integration code owner.
+In this specific example, the integration will be downgraded to “Bronze,” even if it otherwise fully complies with the “Platinum” tier.
+
+## Adjustments to rules contained in each tier
+The world of IoT and all technologies used by Home Assistant are changing at a fast pace.
+Not just in terms of what Home Assistant can support or do, but also the software Home Assistant is built upon is constantly innovated.
```

Comment:
It seems this text hasn't been updated with the suggestions from the ADR PR.

Reply:
Good point, whoops